### PR TITLE
Explain supported branches and regressions better

### PR DESCRIPTION
**Pull request by eternoendless**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 8.x
| Description?  | I enhanced the description for supported branches and what goes where

### CODE REVIEW

File: contribute/contribution-guidelines/supported-branches.md [hunk 1]
```diff
@@ -5,16 +5,29 @@ weight: 30
...
 # Supported branches
 
-Maintainers should only accept contributions for branches which are subject to new releases.
+Maintainers will only accept contributions for branches which are subject to new releases.
 
-Once a minor "dot-zero" version has been released, no new patch releases will be made for previous versions. This means that only the latest minor version patch branch is supported – with the exception of rare cases, like a security bug being found just before or after a minor release is published.
+## Understanding active branches
 
-For example, the `8.0.x` branch is supported until version 8.1 is released. After that, the only supported version branch will be `8.1.x`, and so on.
+Your should target the appropriate branch according to the nature of the change you intend to contribute:
```
Comment (kpodemski) [inline contribute/contribution-guidelines/supported-branches.md]:
```suggestion
You should target the appropriate branch according to the nature of the change you intend to contribute:
```

**Comment by eternoendless** [comment]
Sorry, forgot to repush

File: contribute/contribution-guidelines/supported-branches.md [hunk 2]
```diff
@@ -5,16 +5,35 @@ weight: 30
...
+
+Except for rare cases, a patch version branch is closed the moment the following minor version is released. For example, the `8.0.x` branch is closed once the 8.1 version has been released, giving way to the `8.1.x` branch. 
+
+**Pull Requests targeting closed branches will not be accepted.**
+
+## Bug fixes and patch branches
+
+Patch branches usually only accept critical bug fixes, like security bugs, and **same-version regression** fixes. 
+
+Regressions are bugs that break a feature that was previously working well. If a feature was working well in 8.0 and is broken by a bug introduced in 8.1, then the bug is a regression introduced in 8.1 and should be fixed in the `8.1.x` branch—as long as that branch is open.
```
Comment (mparvazi) [inline contribute/contribution-guidelines/supported-branches.md:36]: Should the dash be here? Line:36

Comment (mparvazi) [inline contribute/contribution-guidelines/supported-branches.md:36]: branch—as long as

the dash

Comment (kpodemski) [inline contribute/contribution-guidelines/supported-branches.md:36]:
```suggestion
Regressions are bugs that break a feature that was previously working well. If a feature was working well in 8.0 and is broken by a bug introduced in 8.1, then the bug is a regression introduced in 8.1 and should be fixed in the `8.1.x` branch - as long as that branch is open.
```

Comment (eternoendless) [reply contribute/contribution-guidelines/supported-branches.md:36]: The em dash can be with or without spaces. I tend to prefer no spaces when using it.

See https://www.thepunctuationguide.com/em-dash.html

Comment (mparvazi) [reply contribute/contribution-guidelines/supported-branches.md:36]: @eternoendless Yes it should be without spaces. Sorry I didn't know about the em dash.

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @eternoendless

**Pull request merged**